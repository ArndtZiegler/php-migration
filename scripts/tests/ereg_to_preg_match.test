# Patterns with special chars
ereg("/[\r\n]/", $str, $m);
eregi("/[\s[:space:]\n]*</td>[\s[:space:]\n]*<td>[\s[:space:]\n]*/", $text, $m);

# With variable pattern, with whitespace
ereg("/" . $_POST['pattern'] . "/", $_POST['string'], $m);
eregi( "/" . $umlaute[$j][0] . "/", $filter[$i], $m);

# Quote confusion
ereg("/ '/", $sql, $m);

# Flag confusion
ereg( '#blub#ui', $in, $m);

# Already preg
ereg( "/blub/i" , $satz, $m );