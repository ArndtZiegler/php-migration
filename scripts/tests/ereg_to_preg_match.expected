# Patterns with special chars
preg_match("/[\r\n]/", $str, $m);
preg_match("/[\s[:space:]\n]*</td>[\s[:space:]\n]*<td>[\s[:space:]\n]*/i", $text, $m);

# With variable pattern, with whitespace
preg_match("/" . $_POST['pattern'] . "/", $_POST['string'], $m);
preg_match( "/" . $umlaute[$j][0] . "/i", $filter[$i], $m);

# Quote confusion
preg_match("/ '/", $sql, $m);

# Flag confusion
preg_match( '#blub#ui', $in, $m);

# Already preg
preg_match( "/blub/i" , $satz, $m );